activerecord/test/cases/associations/eager_test.rb

```diff
@@ -1697,11 +1697,10 @@ def test_preloading_has_many_through_with_custom_scope
       assert_equal 3, comments_collection.size
     end.last
 
-    if current_adapter?(:Mysql2Adapter, :TrilogyAdapter)
-      assert_match(/WHERE `sharded_comments`.`blog_id` IN \(.+\) AND `sharded_comments`.`blog_post_id` IN \(.+\)/, sql)
-    else
-      assert_match(/WHERE "sharded_comments"."blog_id" IN \(.+\) AND "sharded_comments"."blog_post_id" IN \(.+\)/, sql)
-    end
+    c = Sharded::BlogPost.connection
```

Comment:
Can we call this `connection`? Not sure the single letter / short variable name is of much help.

Reply:
I was following the convention used in other tests like https://github.com/rails/rails/blob/d56db8ff17cfdff94954e4b93b965e929df0ee63/activerecord/test/cases/batches_test.rb#L517C7-L517C7 

I will change it to `connection` if needed.

Reply:
Cool, that's fine then. I looked at this one but hadn't checked other files.